src/feature/relay/selftest.c

```diff
@@ -70,7 +70,7 @@ router_reachability_checks_disabled(const or_options_t *options)
  *   - the network is disabled.
  */
 int
-check_whether_orport_reachable(const or_options_t *options)
+router_skip_orport_reachability_check(const or_options_t *options)
```

Comment:
I would suggest a different name -- this one seems to imply that the function _makes us skip_ reachability checks, when it actually tests _whether we are skipping_ those checks.  Maybe `router_should_skip...` or `router_is_skipping...`?

Reply:
I have made the names longer and hopefully clearer in 05c56ae455de1f8e1dc9816ab9dcd7f0cdc2c994.